packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -21,7 +28,8 @@ const region = getEnvOrFail("AWS_REGION");
 const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
 // converter config
 const PDFConvertTimeout = getEnvOrFail("PDF_CONVERT_TIMEOUT_MS");
-const GRACEFUL_SHUTDOWN_ALLOWANCE_MS = 3_000;
+const GRACEFUL_SHUTDOWN_ALLOWANCE = dayjs.duration({ seconds: 3 });
+const PDF_CONTENT_LOAD_ALLOWANCE = dayjs.duration({ seconds: 2.5 });
```

Comment:
Just to be sure, I think we should validate these with the larger files we have on S3 and/or the ones we got failures on Slack/Sentry.